plugin/src/migrations/RedeemSai.js

```diff
@@ -0,0 +1,23 @@
+import { DAI } from "..";
+
+export default class RedeemSai {
+  constructor(manager) {
+    this._manager = manager;
+    this._tap = this._manager.get('smartContract').getContract('SAI_TAP');
+    return this;
+  }
+
+  off() {
+    return this._tap.off();
+  }
+
+  async getRate() {
+    const fix = await this._tap.fix();
+    return fix / Math.pow(10, 27);
```

Comment:
just fyi, you're losing precision here @ethanbennett. this isn't a blocker since you're only using the value for display purposes, but we should get into the habit of only using BigNumber/Currency so that we don't create bugs when it does matter.